indico/modules/events/registration/models/registrations.py

```diff
@@ -105,6 +105,16 @@ class RegistrationData(db.Model):
         JSON,
         nullable=False
     )
+    #: file contents for a file field
+    file = db.Column(
+        db.LargeBinary,
+        nullable=False
+    )
+    #: metadata of the uploaded file
+    file_metadata = db.Column(
+        JSON,
+        nullable=False
```

Comment:
Here it's fine since it stores a json `null` that way and we do this in pretty much all other places with json fields, too